frontend/src/components/ShootAccessRestrictions/GAccessRestrictions.vue

```diff
@@ -6,195 +6,115 @@ SPDX-License-Identifier: Apache-2.0
 
 <template>
   <div
-    v-if="definitions"
+    v-if="accessRestrictionDefinitions"
```

Comment:
```suggestion
    v-if="accessRestrictionDefinitions?.length"
```
Array is empty in case region without AR support gets selected. v-else div does not show up in this case

Reply:
✔️